autoload/denite/util.vim

```diff
@@ -201,12 +201,16 @@ function! denite#util#delete_buffer(command, bufnr) abort
 endfunction
 
 function! denite#util#input(prompt, text, completion) abort
-    try
-        return input(a:prompt, a:text, a:completion)
-    catch
-        " ignore the errors
-        return ''
-    endtry
+  try
+    if a:completion !=# ''
+      return input(a:prompt, a:text, a:completion)
+    else
+      return input(a:prompt, a:text)
+    endif
+  catch
+    " ignore the errors
+    return ''
+  endtry
```

Comment:
The old implementation.
https://github.com/Shougo/denite.nvim/pull/722/commits/4af5416f58573f166db918a7f9dd4e8f7b4898eb#diff-310bf3f97f2cc2a2fc184f4668e968b9L192